pages/en/lb4/Preparing-the-API-for-consumption.md

```diff
@@ -14,11 +14,23 @@ This article continues off from [Implmenting features](./Implementing-features.
 
 ## Preparing your API for consumption
 
-{% include content/tbd.html %}
+### Interacting with you API
 
-- Documentation
-- Public endpoint for the swagger spec (for discovery)
-- more?
+When your application is run, its documentation is generated into `swagger.json` file and can be accessed from your server through `/swagger.json`. You can also interact with the API endpoints defined in `swagger.json` file through Swagger UI (`/swagger-ui`), which allows users to directly interact with the documented API endpoints of the application.
```

Comment:
You can also mention that we provide the OpenAPI spec as well and in both yaml and json flavours for both versions.

Reply:
I wouldn't yet do that; we haven't confirmed OpenAPI support yet.

Until we've finalized our plans, you can leave that part out.